src/screens/charging-stations/connector-details/ChargingStationConnectorDetails.tsx

```diff
@@ -727,27 +754,59 @@ export default class ChargingStationConnectorDetails extends BaseAutoRefreshScre
           <View style={style.noButtonStopTransaction} />
         )}
         {/* Details */}
-        <ScrollView
-          contentContainerStyle={style.scrollViewContainer}
-          refreshControl={<RefreshControl refreshing={this.state.refreshing} onRefresh={this.manualRefresh} />}>
-          <View style={style.rowContainer}>
-            {this.renderConnectorStatus(style)}
-            {this.renderUserInfo(style)}
-          </View>
-          <View style={style.rowContainer}>
-            {this.renderInstantPower(style)}
-            {this.renderTotalConsumption(style)}
-          </View>
-          <View style={style.rowContainer}>
-            {this.renderElapsedTime(style)}
-            {this.renderInactivity(style)}
-          </View>
-          <View style={style.rowContainer}>
-            {this.renderBatteryLevel(style)}
-            {isPricingActive ? this.renderPrice(style) : <View style={style.columnContainer} />}
-          </View>
-        </ScrollView>
+        {connector?.status === ChargePointStatus.AVAILABLE ? (
+          <ScrollView
+            contentContainerStyle={style.scrollViewContainer}
+            refreshControl={
+              isUserModalVisible ? null : <RefreshControl refreshing={this.state.refreshing} onRefresh={this.manualRefresh} />
+            }>
+            <View style={style.rowContainer}>{this.renderConnectorStatus(style)}</View>
```

Comment:
To reduce the top margin/padding
![image](https://user-images.githubusercontent.com/25266945/116949495-ac37dc00-ac82-11eb-87dd-52e34c633028.png)

Reply:
Done